src/main/java/com/mojang/brigadier/Command.java

```diff
@@ -6,9 +6,30 @@
 import com.mojang.brigadier.context.CommandContext;
 import com.mojang.brigadier.exceptions.CommandSyntaxException;
 
+/**
+ * The functional interface actually representing a command to execute.
+ * <p>
```

Comment:
See https://github.com/Mojang/brigadier/blob/master/src/main/java/com/mojang/brigadier/CommandDispatcher.java#L70 - wrap in `<p>` like that.

Reply:
Just as a note to anyone reading, this is not the only occurrence of `<p>` being used like this.

Reply:
It is one out of 77 places where it is used like that, the only times where it is used differently is in CommandDispatcher. It isn't well formed HTML, but reads quite nice IMHO. I just forgot to adjust the existing ones.

I can adjust it to match the way it was done in CommandDispatcher instead though.

Reply:
While `</p>` *can* be omitted, I personally find it nicer to format as done in `CommandDispatcher` - it is nice to see "it starts here and ends here" instead of just "it starts here and automatically ends... here".